### PR TITLE
AGP-130: Aragon Mesh Sponsorship

### PR DESCRIPTION
**Pull request by 0xGabi**
Proposal to Sponsorship Aragon Mesh.

TODO:
- [x] Use the Agent as Vault: `0x3baa6420a4c683b497fa392ab9ebd89f2c041fe2`

### CODE REVIEW

File: AGPs/AGP-120.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+AGP: 120
+Title:
```
Comment (john-light) [inline AGPs/AGP-120.md]: AGP needs a title

**Comment by john-light** [comment]
@0xGabi your PR description says:

> TODO
> Use the Agent as Vault: 0x3baa6420a4c683b497fa392ab9ebd89f2c041fe2

Is this change still needed to the AGP to finalize it?

File: AGPs/AGP-120.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Our mission is to improve the developer experience as a whole, which includes easing the on-boarding process of new developers that wish to work on the Aragon core infrastructure & SDK. This AGP aims to compensate those devs.
+
+In this new endeavor we are looking forward to formalizing and improving the current process. We have been keeping track of milestones in this GitHub repository: [AragonMesh/team](https://github.com/AragonMesh/team). As part of improving the current process we will like to continue experimenting with the [Projects app](https://github.com/AutarkLabs/open-enterprise/blob/dev/docs/GETTING_STARTED.md#install-the-projects-app) of the Open Enterprise suite and bounty-based compensation.
```
Comment (john-light) [inline AGPs/AGP-120.md]:
```suggestion
In this new endeavor we are looking forward to formalizing and improving the current process. We have been keeping track of milestones in this GitHub repository: [AragonMesh/team](https://github.com/AragonMesh/team). As part of improving the current process we would like to continue experimenting with the [Projects app](https://github.com/AutarkLabs/open-enterprise/blob/dev/docs/GETTING_STARTED.md#install-the-projects-app) of the Open Enterprise suite and bounty-based compensation.
```

File: AGPs/AGP-120.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Amount of the transfers
+
+30,000 DAI
+
+5,000 ANT
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+
+Quarterly transfers of 30,000 DAI minus unused funds from the previous quarter if any. This is the amount to be debited from the quarterly Finance track budget.
```
Comment (john-light) [inline AGPs/AGP-120.md]: Is this meant to indicate that only the DAI portion of the specified transfers is to be recurring quarterly?

File: AGPs/AGP-120.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+The transfer will be made after each ANV has completed.
+
+## Purpose of the transfer
+
+This sponsorship is to keep a continue stream of funding to support new developers wanting to contribute to core infrastructure of Aragon. ANT will be used as [reward](https://www.notion.so/Reward-system-ce1b3893f8a3496b801d88675f9f7250) mechanism.
```
Comment (john-light) [inline AGPs/AGP-120.md]:
```suggestion
This sponsorship is to keep a continue stream of funding to support new developers wanting to contribute to core infrastructure of Aragon. ANT will be used as a [reward](https://www.notion.so/Reward-system-ce1b3893f8a3496b801d88675f9f7250) mechanism.
```

File: AGPs/AGP-120.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+The founds will be controller by token holders of the `MESH` token.
```
Comment (john-light) [inline AGPs/AGP-120.md]:
```suggestion
The funds will be controlled by token holders of the `MESH` token. 
```

File: AGPs/AGP-120.md (hunk 3 above)
Comment (0xGabi) [reply AGPs/AGP-120.md]: Good catch, I added ANT as well to recurring transfer.

**Comment by 0xGabi** [comment]
This AGP is final and ready to move to Stage V.

**Pull request merged**